Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -248,6 +248,11 @@ class UserTextInputAlert {
                         attributes: [.foregroundColor: UIColor.placeholderText])
                 } else {
                     textField.placeholder = placeholder
+                    
+                    if PrivateBrowsingManager.shared.isPrivateBrowsing {
+                        UITextField.appearance(whenContainedInInstancesOf: [UIAlertController.self]).defaultTextAttributes =
+                            [.foregroundColor: UIColor.black]
+                    }
```

Comment:
What happens if you switch back to normal browsing? Wouldn't the text be stuck as black?

Reply:
Good question and the answer is I tested it with a red color and it doesnt stuck.

Reply:
This should be placed in `AppearanceExtensions` and maybe assign values to both dark and light modes? needs testing though

Reply:
So in iOS 12 doesnt have difference in light and dark mode also this change will not get effected regarding to private or not. So moved the change into Appearance Extensions for iOS12